seatunnel-common/src/main/java/io/github/interestinglab/waterdrop/common/config/Common.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.github.interestinglab.waterdrop.common.config;
+
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+public class Common {
+
+    private static final List<String> ALLOWED_MODES = Arrays.asList("client", "cluster");
+
+    private static Optional<String> MODE = Optional.empty();
+
+    public static boolean isModeAllowed(String mode) {
+        return ALLOWED_MODES.stream().anyMatch(m -> mode.toLowerCase().equals(m));
```

Comment:
return ALLOWED_MODES.stream().anyMatch(m -> mode.equalsIgnoreCase(m));

wouldn't it be better?

Reply:
return ALLOWED_MODES.contains(mode.toLowerCase());
wouldn't it be better?

Reply:
good job!